doc/source/conf.py

```diff
@@ -138,6 +141,8 @@
 with open("links.rst") as f:
     rst_epilog += f.read()
 
+current_mechanical_version = next(iter(SUPPORTED_MECHANICAL_EMBEDDING_VERSIONS_WINDOWS.keys()))
```

Comment:
Why this next(iter(...))? - just access the key in the list you are interested. In you case ``SUPPORTED_MECHANICAL_EMBEDDING_VERSIONS_WINDOWS.keys()[0]``
```suggestion
current_mechanical_version = SUPPORTED_MECHANICAL_EMBEDDING_VERSIONS_WINDOWS.keys()[0]
```

Reply:
@RobPasMue I tried it first but `TypeError: 'dict_keys' object is not subscriptable`